### PR TITLE
Bump up hvpa-controller version to v0.2.0

### PR DESCRIPTION
**Pull request by ggaurav10**
**What this PR does / why we need it**:
Bump up hvpa-controller version to v0.2.0:
- Introduces new scale mode: "MaintenanceWindow"
- Scales `limits` along with `requests`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Will remove the WIP label once new release of HVPA is created

**Release note**:

```noteworthy operator
The shoot etcds are now scaled down only during maintenance time window.
```

```noteworthy operator
The resource limits for etcds and the kube-apiservers of shoots are now scaled along with the resource requests.
```

``` improvement operator github.com/gardener/hvpa-controller #53 @ggaurav10
Handled case when maintenance time window is missing
```

``` improvement operator github.com/gardener/hvpa-controller #51 @ggaurav10
Enhanced hvpa controller logs to include details about the reconciling object.
```

``` improvement operator github.com/gardener/hvpa-controller #50 @ggaurav10
HVPA now supports setting scale mode to `MaintenanceWindow`. When this mode is set, scaling happens only during user-defined maintenance time window.
Currently, this mode is supported only for vertical scaling. Support for horizontal scaling is dependent on implementation of `scale` subresource in HVPA
```

``` improvement operator github.com/gardener/hvpa-controller #49 @ggaurav10
Added support of limits scaling
```

``` improvement operator github.com/gardener/hvpa-controller #48 @ggaurav10
Override vpaWeight if HPA is not maxed out and oomkill still happens
```

``` improvement operator github.com/gardener/hvpa-controller #47 @ialidzhikov
The release tags from now are prefixed with `v`.
```

``` improvement operator github.com/gardener/hvpa-controller #46 @amshuman-kr
Aggregated metrics are always initialised. Detailed metrics are initialised as soon as the corresponding resource is reconciled.
```

``` improvement operator github.com/gardener/hvpa-controller #42 @ggaurav10
Change default port to `9569`
```

### CODE REVIEW

File: charts/seed-controlplane/charts/etcd/templates/etcd-hvpa.yaml
Comment (timebertt) [inline charts/seed-controlplane/charts/etcd/templates/etcd-hvpa.yaml]: What do you think of making theses values configurable via chart values and putting the defaults in `values.yaml`?

File: charts/seed-controlplane/charts/etcd/values.yaml
Comment (timebertt) [inline charts/seed-controlplane/charts/etcd/values.yaml:42]: Can you provide some example values here to show, what format these values would have?
Because AFAIK Shoots will always have a (at least random) maintenanceWindow, so we will probably always pass some values here, right?

File: charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver-hvpa.yaml
Comment (timebertt) [inline charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver-hvpa.yaml]: Can we put these values also to `values.yaml`?

File: charts/images.yaml
Comment (timebertt) [inline charts/images.yaml]:
```suggestion
  tag: "v0.2.0"
```

Comment (timebertt) [inline charts/images.yaml]: I guess with this upgrade all other Hvpa manifests should also be adapted, right?
Can you please adapt `charts/gardener/controlplane/charts/runtime/templates/apiserver/hvpa.yaml` accordingly?
Please also decrease the limits as discussed [here](https://github.com/gardener/gardener/pull/1880#discussion_r374526808)

File: charts/seed-controlplane/charts/etcd/values.yaml
Comment (ggaurav10) [reply charts/seed-controlplane/charts/etcd/values.yaml:42]: Thanks @tim-ebert for the review. Is that ok if we provide the sample data as a comment, so that even if we chose to remove maintenanceWindow requirement in the future, it doesn't impact here?

Comment (timebertt) [reply charts/seed-controlplane/charts/etcd/values.yaml:42]: Yes, that's ok for me

**Review by amshuman-kr — APPROVED** [review]
LGTM

**Review by timebertt — COMMENTED** [review]
/lgtm
Waiting for the release now.
Let's keep an eye on this and observe it for some time once it's merged.

**Comment by ggaurav10** [comment]
hvpa 0.2.0 is released. This PR is ready for the merge.
Thanks to all the reviewers :)

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**
